api/src/main/java/org/eclipse/microprofile/rest/client/RestClientBuilder.java

```diff
@@ -50,11 +51,36 @@ static RestClientBuilder newBuilder() {
      * <code>http://my-service:8080/service/api</code> in addition to any
      * <code>@Path</code> annotations included on the method.
      *
+     * The {@link #baseUri(URI)} method should be preferred over this method.
```

Comment:
Saying that `baseUri` is preferred is misleading.  Are you saying that using the following chain, the value of `baseUri` is used?

`newBuilder().baseUri(someUri).baseUrl(someUrl).proxy(someClass)`

Reply:
Agreed - it is not good wording.  In that case, no - the last call (to either baseUri or baseUrl) wins.  I've got a TCK test that confirms that, but I will work on the wording to indicate that baseUri is only "preferred" in that URIs are preferred to URLs - in the google groups, it was mentioned that URLs have some blocking methods where some methods will result in network activity - see http://blog.markfeeney.com/2010/11/java-uri-vs-url.html for one instance.